packages/core/src/fhir-deduplication/resources/medication-request.ts

```diff
@@ -0,0 +1,52 @@
+import { MedicationRequest } from "@medplum/fhirtypes";
+import { combineResources, fillMaps } from "../shared";
+
+export function deduplicateMedRequests(medications: MedicationRequest[]): {
+  combinedMedRequests: MedicationRequest[];
+  refReplacementMap: Map<string, string[]>;
+} {
+  const { medRequestsMap, remainingMedRequests, refReplacementMap } =
+    groupSameMedRequests(medications);
+  return {
+    combinedMedRequests: combineResources({
+      combinedMaps: [medRequestsMap],
+      remainingResources: remainingMedRequests,
+    }),
+    refReplacementMap,
+  };
+}
+
+/**
+ * Approach:
+ * 1 map, where the key is made of:
+ * - medicationReference ID
+ * - status
+ */
+export function groupSameMedRequests(medRequests: MedicationRequest[]): {
```

Comment:
you noted that for medRequests, we aren't even mapping the dosage. We aren't mapping [dates](https://github.com/metriport/metriport/blob/a83ff7f9394dde4508a67349d840cba260cac326/packages/fhir-converter/src/templates/cda/Resources/MedicationRequest.hbs#L6) either! I don't think we can responsibly do any deduplication without any meaningful clinical data currently inside the MedicationRequest resource! 

We can leave this in the PR for now but the MedicationRequest resource is near almost useless right now without a date. 
Can we make a high prio ticket to add that and then revisit the medicationRequest deduplication logic?

Reply:
Yea, I'll create the ticket and see if I can quickly put a PR together for htis